CONTRIBUTING.md

```diff
@@ -75,29 +80,89 @@ Note that the FLS ID should be filled according to the FLS paragraph ID for whic
 
 You would then pull `fls_4rhjpdu4zfqj` to place in the FLS ID field.
 
-### 2) Create a Draft with a Member
 
-A member of the Coding Guidelines Subcommittee should get you a first review with some feedback within 14 days of submission. You'll work with one or more members to flesh out the concept and ensure the guideline is well prepared.
+### 2) Guideline Generated as a Comment
+
+A GitHub Action will fire, adding a comment to your newly created issue with
+the contents of the coding guideline prepared written out correctly
+in reStructuredText.
+
+Note that if you later update the body of the coding guideline issue this will
+fire the GitHub Action again and update the original comment with the new
+contents converted to reStructuredText.
+
+### 3) Create a Draft with a Member
+
+Within 14 days of your submission, a member of the Coding Guidelines Subcommittee should give you a first review. You'll work with them (and other members) to flesh out the concept and ensure the guideline is well prepared for a Pull Request.
+
+### 4) Create the PR
+
+Once an issue has been well-developed enough, a subcommittee member will mark
+the issue with the label `sign-off: create pr` to denote that you
+should create a Pull Request.
```

Comment:
Let's add a definition of "well-developed" here:

> Here's a list of recommended prerequisites that shall be fulfilled before turning an issue into a PR:
> * The new rule isn't already covered by another rule or, in case there is already another rule, the rules need to be linked and categorized
> * All sections contain some content
> 
> As soon as these are fulfilled, the issue shall be converted into a PR by a subcommittee member by marking the issue with the label `sign-off: create pr` to denote that you should create a Pull Request. Further discussion about the amount and correctness of the content shall then be done on the Pull Request and the Issue gets closed.